constants/util.go

```diff
@@ -69,10 +70,11 @@ var (
 			DHubURL:          "docker.io/bcgsc/abyss", // https://github.com/bcgsc/abyss
 			OutputDir:        AbyssOut,
 			AssemblyFileName: "final-contigs.fa",
-			Comm: func() []string {
+			Comm: func(threads int) []string {
 				return []string{
 					`k=25`,
 					`name=final`,
+					fmt.Sprintf("j=%s", string(threads)),
 					fmt.Sprintf("in='%s'", RawSeqIn),
 					fmt.Sprintf("--directory=%s", path.Join(BaseOut, AbyssOut)),
```

Comment:
Why use `j`, not `np`? According to the [docs](https://github.com/bcgsc/abyss#parallel-processing), we should use `np=<num_thread>`, rather than `j`.